validator/rpc/standard_api.go

```diff
@@ -98,6 +103,18 @@ func (s *Server) ImportKeystores(
 			return &ethpbservice.ImportKeystoresResponse{Statuses: statuses}, nil
 		}
 	}
+	if len(req.Passwords) == 0 {
+		req.Passwords = make([]string, len(req.Keystores))
+	}
+	if len(req.Passwords) != len(req.Keystores) {
```

Comment:
is there a better way to do this?

Reply:
Probably can fix this by simply checking
```
if len(a) > len(b) {
  a = a[:len(b)]
}
```
I would also recommend we add a comment to the case when len is 0 and to this case explaining why this is a safe thing to do